app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -128,7 +130,7 @@ class HostPreloadedAttributesDecorator < SimpleDelegator
       def initialize(host, connection)
         @obj = host
         @nics = connection.follow_link(host.nics)
-        @statistics = connection.follow_link(host.statistics)
+        @statistics = connection.follow_link(host.statistics) if host.statistics
```

Comment:
This check won't work. Host.statistics returns an array of Statistic. The follow_link method expects to have href attribute on the object passed which array do no have. It seems that we need to iterate over host.statistics and call follow_link for each statistic there is.